tests/res/mocks/libs/bash_script.py

```diff
@@ -43,12 +45,12 @@ def generate_mock(output_path: Path, files_to_save: List[Path], stdout: Optional
         if stdout:
             with open(stdout, 'rb') as f:
                 stdout_content = f.read()
-                stdout_content = base64.standard_b64encode(stdout_content).decode("utf-8")
-                script.write(__STDOUT_TEMPLATE.format(stdout=stdout_content))
+                base64_encoded_content = base64.standard_b64encode(stdout_content).decode("utf-8")
+                script.write(__STDOUT_TEMPLATE.format(stdout=base64_encoded_content))
         if stderr:
             with open(stderr, 'rb') as f:
                 stderr_content = f.read()
-                stderr_content = base64.standard_b64encode(stderr_content).decode("utf-8")
-                script.write(__STDERR_TEMPLATE.format(stderr=stderr_content))
+                base64_encoded_content = base64.standard_b64encode(stderr_content).decode("utf-8")
+                script.write(__STDERR_TEMPLATE.format(stderr=base64_encoded_content))
```

Comment:
/reviewpad explain

Reply:
I added another variable. The previous one was of type bytes and this one is a string (utf-8 and base64 encoded representation of stdout file)

```
            with open(stdout, 'rb') as f:
                # stdout as bytes
                stdout_content = f.read()
                # stdout as utf-8 encoded base64 string
                base64_encoded_content = base64.standard_b64encode(stderr_content).decode("utf-8")
                script.write(__STDOUT_TEMPLATE.format(stdout=base64_encoded_content))
```

I can undo this change if it doesn't add value, or perhaps rename **stdout_content** to **stdout_byte**

Reply:
@hendzeld that is OK, I was just testing reviewpad capabilities